edx_name_affirmation/models.py

```diff
@@ -35,3 +35,18 @@ class Meta:
         """ Meta class for this Django model """
         db_table = 'nameaffirmation_verifiedname'
         verbose_name = 'verified name'
+
+
+class VerifiedNameConfig(TimeStampedModel):
```

Comment:
I thought this model should be a config model. See https://github.com/edx/django-config-models

Reply:
I had gone off of this draft PR - https://github.com/edx/edx-name-affirmation/compare/bseverino/verified-name-config. I didn't know that the edx-config-model existed so I will look into that!

Reply:
^ Didn't know we had that, either 🤯